core-aam/index.html

```diff
@@ -5711,6 +5711,48 @@ <h4>Not Mapped</h4>
               >. Its absence not only indicates that the accessible object is not grabbed, but further defines it as not grab-able. These cases are marked as "Not mapped" without an asterisk.
             </p>
           </section>
+          <h4 id="ariaActions"><code>aria-actions</code></h4>
+          <table aria-labelledby="ariaActions">
+            <tbody>
+              <tr>
+                <th>ARIA Specification</th>
+                <td>
+                  <a class="property-reference" href="#aria-actions"><code>aria-actions</code></a>
+                </td>
+              </tr>
+              <tr>
+                <th>MSAA + IAccessible2</th>
+                <td>
+                  <span class="property"
+                    >Interface: <code>IAccessibleAction</code>, using the accessible nodes matching `aria-actions` IDREFs, if the referenced objects are in the accessibility tree</span
+                  ><br />
+                  <span class="property">Object Attribute: <code>aria-actions</code> &lt;value&gt;</span>
+                </td>
+              </tr>
+              <tr>
+                <th><abbr title="User Interface Automation">UIA</abbr></th>
+                <td>
+                  <span class="property"
+                    >Custom Property: <code>AccessibleActions</code>: element array of accessible nodes matching IDREFs, if the referenced objects are in the accessibility tree</span
+                  ><br />
+                  <span class="property">Object Attribute: <code>aria-actions</code> &lt;value&gt;</span>
+                </td>
+              </tr>
+              <tr>
+                <th><abbr title="Accessibility Toolkit">ATK</abbr>/<abbr title="Assistive Technology-Service Provider Interface">AT-SPI</abbr></th>
+                <td>
+                  <span class="relation">Relation: <code>RELATION_DETAILS</code> points to accessible nodes matching IDREFs, if the referenced objects are in the accessibility tree</span><br />
+                  <span class="relation">Reverse Relation: <code>RELATION_DETAILS_FOR</code> points to element</span>
+                </td>
+              </tr>
+              <tr>
+                <th><abbr title="macOS Accessibility Protocol">AX API</abbr></th>
+                <td>
+                  <span class="property">Implement as <a href="https://developer.apple.com/documentation/uikit/uiaccessibilitycustomaction">custom action</a></span>
```

Comment:
As I understand it, UIAccessibilityCustomAction (as linked here) can only be used with Mac Catalyst. Browsers aren't Mac Catalyst apps, so I don't think they can use this. We need the equivalent for the protocol used by Mac desktop apps.

Reply:
tagging in @cookiecrook for this one, since he wrote the wording for it 😊

Reply:
Yeah, I think we want the macOS-equivalent here, which is [NSAccessibilityCustomAction](https://developer.apple.com/documentation/appkit/nsaccessibilitycustomaction?language=objc), which should be returned as part of an [accessibilityCustomActions](https://developer.apple.com/documentation/appkit/nsaccessibility-c.protocol/accessibilitycustomactions?language=objc) implementation.